### DISCUSSION TITLE
Help trying to contribute

### DISCUSSION THREAD

**Discussion by Shock-1** [ros2-rust/ros2_rust#415] [Q&A]
Hey, i was having some difficulties trying to find how to contribute to this project. I was looking into the issues but a lot of them seem open while implementations seem to have landed in some way. I also wanted to know which features need some work or require contributions in a roadmap sort of way. I tried the matrix link as well but it seems to be broken as it ends in infinitely loading page for me. Thanks in advance!!

**Comment by jhdcs** [comment]
Hm. The matrix link being broken is a bit of an issue. Can you try this one? https://matrix.to/#/!AiYXdTWExqamTdatBl:matrix.org?via=matrix.org&via=uni-osnabrueck.de

If that works, I'll have to update the post where you found the matrix link so that other people can follow and join if they need to.

Also, I think the biggest feature that would be the most useful would be ROS 2 timers. Timers are used by all sorts of ROS 2 programs, but we don't have a working implementation just yet...
